parlai/tasks/integration_tests/agents.py

```diff
@@ -283,6 +301,39 @@ def setup_data(self, fold):
             yield (t, a), e
 
 
+class RepeatWordsTeacher(NocandidateTeacher):
+    """
+    Each input/output pair is a word repeated n times.
+
+    Useful for testing beam-blocking.
+    """
+
+    def __init__(
+        self,
+        opt,
+        shared=None,
+        vocab_size=VOCAB_SIZE,
+        example_size=EXAMPLE_SIZE,
+        num_candidates=NUM_CANDIDATES,
+        num_train=NUM_TRAIN,
+        num_test=NUM_TEST,
+    ):
+        # Set sizes so that we have appropriate number of examples (700)
+        opt['integration_vocab_size'] = 70
+        opt['integration_example_size'] = 11
```

Comment:
why do you go out of your way to make these opts when you can just override them in the call to super()?

Reply:
good point - i'll just make the call to super with overridden values